sample_xblocks/basic/problem.py

```diff
@@ -117,13 +117,29 @@ def student_view(self, context=None):
                     }
                 }
 
-                function handleCheckResults(results) {
-                    $.each(results.submitResults || {}, function(input, result) {
-                        callIfExists(runtime.childMap(element, input), 'handleSubmit', result);
-                    });
-                    $.each(results.checkResults || {}, function(checker, result) {
-                        callIfExists(runtime.childMap(element, checker), 'handleCheck', result);
-                    });
+                function handleCheckResults(data) {
+                    return function(results) {
```

Comment:
Why wrap this function?  You can remove the inner return function(results) function here.